tests/RobotFramework/tests/cumulocity/remote-access/test_remote_access.robot

```diff
@@ -16,14 +16,24 @@ Install/uninstall c8y-remote-access-plugin
 
     # Check that a command is used instead of an explicit path #3111
     ${executable}=    Execute Command
-    ...    cmd=grep "^command" /etc/tedge/operations/c8y/c8y_RemoteAccessConnect | cut -d= -f2-
+    ...    cmd=grep "command" /etc/tedge/operations/c8y/c8y_RemoteAccessConnect | cut -d= -f2-
     ...    strip=${True}
     ...    timeout=5
     Should Match Regexp    ${executable}    pattern=^"c8y-remote-access-plugin\\b
 
     Execute Command    dpkg -r c8y-remote-access-plugin
     File Should Not Exist    /etc/tedge/operations/c8y/c8y_RemoteAccessConnect
 
+Init c8y-remote-access-plugin with the custom user and group
+    Device Should Have Installed Software    c8y-remote-access-plugin
+
+    Execute Command    sudo c8y-remote-access-plugin --init --user petertest --group petertest
```

Comment:
Minor suggestion. Using different values for --user and --group will potentially detect more bugs.

```suggestion
    Execute Command    sudo c8y-remote-access-plugin --init --user peter --group test
```

Reply:
In this case I used the user/group that are present in the test container, so I would need to add new one to test different user and group. Besides, this test is just another variation of what we do with `tedge init` and I think it is sufficient for change that was implemented.

Reply:
ah ok. Then ignore my comment :)